apps/src/netsim/netsim.js

```diff
@@ -219,7 +219,7 @@ NetSim.prototype.init = function(config) {
     return page({
       data: {
         localeDirection: getStore().getState().isRtl ? 'rtl' : 'ltr',
-        instructions: this.level.instructions
+        instructions: this.level.shortInstructions
```

Comment:
Are shortInstructions guaranteed to be available?  I think in the level I tested this morning there were only longInstructions... but it was one of the /s/netsim levels so maybe that's not typical.

Reply:
I don't think it's guaranteed to be there. Is there something you think we should do if there are no short instructions? Or is it fine to just leave that space blank?

Reply:
Now that I think about it, we can probably leave the space blank and lean on level designers to make sure short instructions are provided for NetSim levels.